app/frontends/ogc-coverages/util/parameter-parsing.ts

```diff
@@ -160,6 +160,44 @@ function _getDimensionName(value: string): string {
   }
 }
 
+/**
+ * Parses the provided point parameters and ensures they are valid, throwing an error message
+ * if not
+ *
+ * @param values - An array of all the specified point parameters from the request
+ * @param dimConfig - A mapping of dimension names to min, max, and data type values,
+ *   see `dimensionInfo` (the default value) in this file.  Usually should not be specified,
+ *   except for testing.
+ * @returns An array with two elements corresponding to [longitude, latitude]
+ * @throws ParameterParseError - if a subset parameter cannot be parsed, has unrecognized
+ *   axis names, or is otherwise invalid
+ */
+export function parsePointParam(
+  values: string,
+  dimConfig: DimensionConfig = dimensionConfig,
+): number[] {
+  let results;
+  let coordinate, coordinates;
+  if (values !== undefined){
+    coordinates = values;
+    if ( coordinates.length !== 2 )
+      throw new ParameterParseError(`wrong number of spatial coordinates provided in "${values}"`);
+
+    results = ['lon', 'lat'].map( (dimName, idx): number => {
+      const dim = dimConfig[dimName];
+      coordinate = coordinates[idx];
```

Comment:
How does this work? It seems like you took a string parameter, `values`, and tried to coerce it to an array somehow?

Reply:
Fixed by correcting the parameter type.